addons/website_event/models/event_event.py

```diff
@@ -35,8 +35,12 @@ def _default_cover_properties(self):
         })
         return res
 
+    def _default_share_event(self):
+        return self.env['ir.ui.view']._render_template('event.social_share')
+
     # description
     subtitle = fields.Char('Event Subtitle', translate=True)
+    share_event = fields.Html(sanitize_attributes=False, default=_default_share_event)
```

Comment:
Hum :thinking:
Adding a field just for the social info is a bit overkill and when we take into consideration the size it will take in DB, it's a bit too much imo.
Also seems like the widget we use does not really support any changes easily. The links can't be updated, it's just used to share the default media of the client and not the social media regarding the event. So I suppose it makes sense that it falls in the description column template which is shared between all events.

If the client wants to share some social media related to the event, he can use the social media widget which allows a lot more customization when he update its event description.

Reply:
> it falls in the description column template which is shared between all events.

Agreed, I thought, maybe there are some snippets user just want for that specific event.. like while we are allowing to edit the share block, and give space to add new widgets but some event doesn't need that new widget there... anyways you do have point it will take some space in DB. :upside_down_face: